app/services/container_service_mixin.rb

```diff
@@ -63,16 +64,16 @@ def utilization_data(used_cpu, total_cpu, used_mem, total_mem)
     if used_cpu.any?
       {
         :cpu => {
-          :used  => used_cpu.values.last.round,
-          :total => total_cpu.values.last.round,
+          :used  => (used_cpu.values.last || 0).round(DISPLAY_PRECISION),
+          :total => (total_cpu.values.last || 0).round,
```

Comment:
Explicit argument here too?

Reply:
Yeah, thats total cpu cores. it cant be a fraction.